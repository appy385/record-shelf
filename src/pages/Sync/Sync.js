import React from 'react';
import './Sync.scss';

const Sync = () => (
  <div className="sync-container">
    <span>:((</span>
    <span>Seems a bit empty in here...</span>
    <button type="button" className="sync-button">Sync</button>
  </div>
);

export default Sync;
