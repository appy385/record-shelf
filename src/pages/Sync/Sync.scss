
.sync-container {
    display: flex;
    flex-direction: column;
    align-items: center;
    margin: auto;
    border: 1px solid black;
}

// .sync-button {
//     align-self: center;
// }