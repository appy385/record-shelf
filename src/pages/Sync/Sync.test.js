import React from 'react';
import { render, screen } from '@testing-library/react';
import Sync from './Sync';

describe(Sync.name, () => {
  test('should match snapshot', () => {
    const { container } = render(<Sync />);
    expect(container).toMatchSnapshot();
  });
  test('should have message ', () => {
    render(<Sync />);
    screen.getByText('Seems a bit empty in here...');
    screen.getByText(':((');
  });
  test('should have button sync', () => {
    render(<Sync />);
    const syncButton = screen.getByText('Sync');
    expect(syncButton.tagName).toBe('BUTTON');
  });
});
