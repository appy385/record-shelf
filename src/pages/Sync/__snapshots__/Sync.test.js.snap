// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sync should match snapshot 1`] = `
<div>
  <div
    class="sync-container"
  >
    <span>
      :((
    </span>
    <span>
      Seems a bit empty in here...
    </span>
    <button
      class="sync-button"
      type="button"
    >
      Sync
    </button>
  </div>
</div>
`;
